src/goose/brokers/redis/commands.clj

```diff
@@ -110,6 +110,9 @@
        (filter match?)
        (doall)))
 
+(defn move-in-set [conn src dst elem]
```

Comment:
renamed it to `move-btn-set`.

Reply:
What is btn? I was thinking this could be `update-batch-status-of-job` like `dequeue-and-preserve`, which does the same for lists.

FWIW, Sezal, this isn't nitpick. Bad function names in past have confused future contributors (ahem ahem... ready-queue) 😜

Reply:
I am thinking move-btw-set.
this is just to move an element between two sets

Reply:
@olttwa , it does not make sense to add `batch` context in command namespace. I will rename it to `move-between-set` to make the between clear. 😄